Source/OxyPlot/Series/BoxPlotSeries.cs

```diff
@@ -383,6 +404,35 @@ public override void Render(IRenderContext rc, PlotModel model)
                         rc.DrawEllipse(ellipseRect, fillColor, OxyColors.Undefined, 0);
                     }
                 }
+
+                if (!this.ShowMeanAsDot && !double.IsNaN(item.Mean))
+                {
+                    // Draw the median line
+                    var meanLeft = this.Transform(item.X - halfBoxWidth, item.Mean);
+                    var meanRight = this.Transform(item.X + halfBoxWidth, item.Mean);
+                    rc.DrawClippedLine(
+                        clippingRect,
+                        new[] { meanLeft, meanRight },
+                        0,
+                        strokeColor,
+                        this.StrokeThickness * this.MeanThickness,
+                        LineStyle.Dash.GetDashArray(),
+                        LineJoin.Miter,
+                        true);
+                }
+                else if (!double.IsNaN(item.Mean))
```

Comment:
Yeah you are right, but it makes no sense to change this before we haven't decided what to do about the Line/Maker style stuff overall.